src/vse_sync_pp/analyzers/analyzer.py

```diff
@@ -265,3 +270,193 @@ def explain(self, data):
             'duration': data.iloc[-1].timestamp - data.iloc[0].timestamp,
             'terror': self._statistics(data.terror, 'ns'),
         }
+
+
+class TimeIntervalErrorAnalyzerBase(Analyzer):
+    """Analyze Time Interval Error (also referred to as Wander).
+
+    Derived classes calculate specific Time Interval Error metric focused on measuring
+    the change of Time Error.
+    """
+    locked = frozenset()
+
+    def __init__(self, config):
+        super().__init__(config)
+        # samples in the initial transient period are ignored
+        self._transient = config.parameter('transient-period/s')
+        # minimum test duration for a valid test
+        self._duration = config.parameter('min-test-duration/s')
+        # limit initial tau observation windows from 1 to 9k taus
```

Comment:
From this comment, it looks like it can be up to 9000, while the array contains 10000